strimzi/README.md

```diff
@@ -0,0 +1,237 @@
+    userOperator: {}
+    template:
+      pod:
+        metadata:
+          annotations:
+            ad.datadoghq.com/topic-operator.checks: |
+              {
+                "strimzi": {
+                  "instances":[
+                    {
+                      "topic_operator_endpoint": "http://%%host%%:8080/metrics"
+                    }
+                  ]
+                }
+              }
+            ad.datadoghq.com/user-operator.checks: |
+            {
+              "strimzi": {
+                "instances":[
+                  {
+                    "user_operator_endpoint": "http://%%host%%:8081/metrics"
+                  }
+                ]
+              }
+            } 
+...
+```
+**Note**: Template used as the basis for this example can be found [here][14].
+
+See the [sample strimzi.d/conf.yaml][4] for all available configuration options.
+
+#### Kafka and Zookeeper
+
+The Kafka and Zookeeper components of Strimzi can be monitored using the [Kafka][11] and [Zookeeper][12] checks, respectively. Kafka metrics are collected via JMX. For more information on enabling JMX, please refer to this [link][15]. Here's an example of how to configure the Kafka and Zookeeper checks using Pod annotations:
+```yaml
+apiVersion: kafka.strimzi.io/v1beta2
+kind: Kafka
+metadata:
+  name: my-cluster
+spec:
+  kafka:
+    jmxOptions: {}
+    version: 3.4.0
+    replicas: 1
+    listeners:
+      - name: plain
+        port: 9092
+        type: internal
+        tls: false
+      - name: tls
+        port: 9093
+        type: internal
+        tls: true
+    template:
+      pod:
+        metadata:  
+          annotations:
+            ad.datadoghq.com/kafka.checks: |
+              {
+                "kafka": {
+                  "init_config": {
+                    "is_jmx": true, 
+                    "collect_default_metrics": true, 
+                    "new_gc_metrics": true
+                  },
+                  "instances":[
+                    {
+                      "host": "%%host%%",
+                      "port": "9999"
+                    }
+                  ]
+                }
+              }        
+    config:
+      offsets.topic.replication.factor: 1
+      transaction.state.log.replication.factor: 1
+      transaction.state.log.min.isr: 1
+      inter.broker.protocol.version: "3.4"
+    storage:
+      type: ephemeral
+  zookeeper:
+    replicas: 1
+    storage:
+      type: ephemeral
+    template:
+      pod:
+        metadata:
+          annotations:
+            key1: label3
+            key2: label4
+            ad.datadoghq.com/zookeeper.checks: |
+              {
+                "zk": {
+                  "instances":[
+                    {
+                      "host":"%%host%%","port":"2181"
+                    }
+                  ]
+                }
+              } 
+```
+**Note**: Template used as the basis for this example can be found [here][14].
+
+##### Log collection
```

Comment:
To confirm, is this Log Collection section a subheading of #### Kafka and Zookeeper or ### Configuration? If it's the latter:
```suggestion
#### Log collection
```